azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/CallComposite.java

```diff
@@ -168,6 +209,43 @@ public void removeOnDismissedEventHandler(final CallCompositeEventHandler<CallCo
         configuration.getCallCompositeEventsHandler().removeOnExitEventHandler(handler);
     }
 
+    /**
+     * Add {@link CallCompositeEventHandler}.
+     *
+     * <p> Add a callback for Call Composite Audio Selection Changed Event.
+     * See {@link com.azure.android.communication.ui.calling.models.CallCompositeAudioSelectionChangedEvent}
+     * for values.</p>
+     * <pre>
+     *
+     * &#47;&#47; add audio selection changed handler
+     * callComposite.addOnAudioSelectionChangedEventHandler&#40;event -> {
+     *     &#47;&#47; Process audio selection changed event
+     *     System.out.println&#40;event.getSelectionType&#40;&#41;&#41;;
+     * }&#41;;
+     *
+     * </pre>
+     *
+     * @param eventHandler The {@link CallCompositeEventHandler}.
+     */
+    public void addOnAudioSelectionChangedEventHandler(
```

Comment:
I'm a bit questionable of these events and the naming, since the event only holds a string.

Without a fixed set of values or documentation on values, a developer can't do much with it.

If the strings come from the system, can we map to one of our enums for headset/speaker/bluetooth?

Reply:
As per discussion we will fix for GA.